ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButton: UIButton {
+    let item: CommandBarItem
+
+    override var isHighlighted: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    override var isSelected: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    override var isEnabled: Bool {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    init(item: CommandBarItem, isFixed: Bool = false) {
+        self.item = item
+        self.isFixed = isFixed
+
+        super.init(frame: .zero)
+
+        translatesAutoresizingMaskIntoConstraints = false
+        setImage(item.iconImage, for: .normal)
+
+        accessibilityLabel = item.accessibilityLabel
+        contentEdgeInsets = CommandBarButton.contentEdgeInsets
+
+        if #available(iOS 14.0, *) {
+            menu = item.menu
+            showsMenuAsPrimaryAction = item.showsMenuAsPrimaryAction
+        }
+
+        updateState()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func updateState() {
+        isEnabled = item.isEnabled
+        isSelected = !isFixed && item.isSelected
+    }
+
+    private let isFixed: Bool
+}
+
+private extension CommandBarButton {
+    var selectedTintColor: UIColor {
+        guard let window = window else {
+            return Colors.communicationBlue
+        }
+
+        return Colors.primary(for: window)
+    }
+
+    var selectedBackgroundColor: UIColor {
+        guard let window = window else {
+            return Colors.Palette.communicationBlueTint30.color
+        }
+
+        return Colors.primaryTint30(for: window)
+    }
+
+    func updateStyle() {
+        tintColor = isSelected ? selectedTintColor : CommandBarButton.normalTintColor
+
+        if isFixed {
```

Comment:
if it is a "fixed" button, then there isn't any visual feedback if user long press and hold? no color changes at all?

Reply:
Yeah. That's an issue. Do we have the design spec for this scenario?